Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/analytics/worker/DataDonationAnalyticsOneTimeWorker.kt

```diff
@@ -0,0 +1,51 @@
+package de.rki.coronawarnapp.datadonation.analytics.worker
+
+import android.content.Context
+import androidx.work.CoroutineWorker
+import androidx.work.WorkerParameters
+import dagger.assisted.Assisted
+import dagger.assisted.AssistedFactory
+import dagger.assisted.AssistedInject
+import de.rki.coronawarnapp.datadonation.analytics.Analytics
+import de.rki.coronawarnapp.util.worker.InjectedWorkerFactory
+import de.rki.coronawarnapp.worker.BackgroundConstants
+import timber.log.Timber
+
+/**
+ * One time background Analytics worker
+ *
+ * @see DataDonationAnalyticsScheduler
+ */
+class DataDonationAnalyticsOneTimeWorker @AssistedInject constructor(
```

Comment:
Do we need a onetime worker? Can win with just the periodic worker?

Reply:
Why do they both look the same?

Reply:
currently the randomness was achieved by scheduling one time work every 24 hours which would then schedule periodic work at a random time in the next 23 hours, but i like the idea of just randomly distributing the initial start delay of the periodic work, will remove the one time worker